.pre-commit-config.yaml

```diff
@@ -13,7 +13,7 @@ repos:
       - id: check-executables-have-shebangs
       - id: check-json
       - id: check-merge-conflict
-        exclude: "tests/func/utils/test_strict_yaml.py"
+        exclude: "tests/func/test_dvcfile_validation.py"
```

Comment:
Let's not move the tests please.

Reply:
> Let's not move the tests please.

I apologize for any confusion caused by my previous commit. My intention was to keep the strictyaml module unaware of any details related to dvcfile, which is why I moved all functional tests concerning dvcfile validation to the tests/dvcfile_validation directory. If you're confident that leaving the old name is the best approach, please let me know.

Reply:
In DVC, `dvc.yaml` validation is the only consumer of this. Any other tests could be considered as unit tests and may belong in `tests/unit`. The tests in `test_strictyaml` are graphical rather than acutal validation, so `test_dvcfile_validation` is not a good name either.

Anyway, let's not mix different things in the same PR.